server/src/main/java/org/opensearch/tasks/TaskInfo.java

```diff
@@ -94,7 +96,8 @@ public TaskInfo(
         long runningTimeNanos,
         boolean cancellable,
         TaskId parentTaskId,
-        Map<String, String> headers
+        Map<String, String> headers,
+        Map<String, Long> statsInfo
```

Comment:
We should instead use some `stats` object as value rather than `Long` to help with adding other fields in that object in future for each stats type